types/dygraphs/index.d.ts

```diff
@@ -408,7 +408,7 @@ declare namespace dygraphs {
          * earliest/latest are milliseconds since epoch. If the data for the x-axis is numeric, the
          * values in dateWindow must also be numbers.
          */
-        dateWindow?: number[];
+        dateWindow?: (Date|number)[];
```

Comment:
The comment above this field (and the documentation at http://dygraphs.com/options.html#dateWindow) specifically say that this should be millis since epoch, rather than `Date` objects. `Date` objects do seem to work in this context, but that's more an accident than intentional behavior, and it might lead to subtle bugs. I'd prefer to keep this as `number[]`.

Reply:
Thank you for clarification. Did not know it could cause side effects.